src/app.ts

```diff
@@ -52,51 +48,53 @@ async function initializeApp() {
         .use(LanguageDetector)
         .init({
             backend: {
-                loadPath: path.join(getLocalesPath(), '{{lng}}.json'),
+                loadPath: path.join(getLocalesPath(), "{{lng}}.json"),
             },
-            fallbackLng: 'en',
-            preload: ['en', 'ja'],
-            supportedLngs: ['en', 'ja'],
+            fallbackLng: "en",
+            preload: ["en", "ja"],
+            supportedLngs: ["en", "ja"],
             nonExplicitSupportedLngs: true,
-            load: 'languageOnly',
+            load: "languageOnly",
             debug: false,
             detection: {
-                order: ['header', 'cookie'],
-                lookupHeader: 'accept-language',
-                lookupCookie: 'i18next',
-                caches: ['cookie']
+                order: ["header", "cookie"],
+                lookupHeader: "accept-language",
+                lookupCookie: "i18next",
+                caches: ["cookie"],
             },
             interpolation: {
-                escapeValue: false
-            }
+                escapeValue: false,
+            },
         });
 
     app.use(handle(i18next));
 
     // to Switch language
-    app.use((req, res, next) => {
+    app.use((req, _res, next) => {
         const currentLanguage = i18next.language;
         i18next.changeLanguage(req.language);
         const newLanguage = i18next.language;
-// Uncomment for debugging
-//        console.log('Language Change:', {
-//            header: req.headers['accept-language'],
-//            detected: req.language,
-//            currentLanguage: currentLanguage,
-//            newLanguage: newLanguage
-//        });
+        if (process.env.DEBUG_I18N) {
+            console.log("Language Change:", {
+                header: req.headers["accept-language"],
+                detected: req.language,
+                currentLanguage: currentLanguage,
```

Comment:
eslint didn't like currentLanguage being defined and not used, so i just wrapped it in an env variable
plus bleh commented out code :)